static/app/gettingStartedDocs/javascript/react.tsx

```diff
@@ -163,8 +174,48 @@ const onboarding: OnboardingConfig = {
   ],
 };
 
+const replayOnboarding: OnboardingConfig = {
+  install: () => [
+    {
+      type: StepType.INSTALL,
+      description: tct(
+        'Add the Sentry SDK as a dependency using [codeNpm:npm] or [codeYarn:yarn]. You need a minimum version 7.27.0 of [code:@sentry/react] in order to use Session Replay. You do not need to install any additional packages.',
+        {
+          code: <code />,
+          codeYarn: <code />,
+          codeNpm: <code />,
+        }
+      ),
+      configurations: getInstallConfig(),
+    },
+  ],
+  configure: (params: Params) => [
+    {
+      type: StepType.CONFIGURE,
+      description: getReplayConfigureDescription({
+        link: 'https://docs.sentry.io/platforms/javascript/guides/react/session-replay/',
+      }),
+      configurations: [
+        {
+          code: [
+            {
+              label: 'JavaScript',
+              value: 'javascript',
+              language: 'javascript',
+              code: getSdkSetupSnippet({params, isReplayOnboarding: true}),
```

Comment:
could we overwrite `params. isReplaySelected` here and then we wouldn't need to have the similar `isReplayOnboarding` param?

```suggestion
              code: getSdkSetupSnippet({...params, isReplaySelected: true}),
```

```
const getSdkSetupSnippet = (params: Params) => {...}
```